src/Array.ts

```diff
@@ -89,6 +91,33 @@ Object.defineProperties(Array.prototype, {
 			return found !== undefined ? map(found) : found;
 		},
 	},
+	count: {
+		enumerable: false,
+		writable: true,
+		configurable: true,
+		value: function (search: RegExp | any) {
+			let i: number = 0;
+			this.forEach((element: any) => {
+				if (element === search) {
+					i++;
+					return;
+				}
+				if (typeof search == "function") {
+					if (typeof element === search.toString().split(" ")[1].slice(0, -2).toLowerCase()) {
```

Comment:
what does this do?

Reply:
```js
f (typeof search == "function") {  // typeof [String,Number,Object,Array] == "function"
	if (typeof element === search.toString() // "function String() { [native code] }"
         .split(" ")[1] // "String()"
         .slice(0, -2)  // "String"
         .toLowerCase()) {  // "string"
		i++;
		return;
	}
}
```
allows the usage of Classes such as (String,Number,Object,Array) in the `count()` method.